cli/cmd/plugin/ui/web/tanzu-ui/src/components/aws/cluster-settings-step/ClusterSettings.scss

```diff
@@ -0,0 +1,38 @@
+.cluster-settings-container {
+    padding: 20px;
```

Comment:
We should avoid setting custom padding where possible. If applying spacing to a div, we should try to use the predefined 'cds-layout' padding options found at https://clarity.design/storybook/core/?path=/story/layout-spacing--page

Alternatively, if we have to define some class with explicit padding, we should probably try to use the cds global variables for the padding values if possible: https://clarity.design/storybook/core/?path=/story/foundation-spacing--page

Let's all discuss over zoom

Reply:
css issues will be handled in another PR.